docs/talkback-guidelines.md

```diff
@@ -0,0 +1,401 @@
+
+#### Focus Navigation
+
+In Android, focus is provided to views that require user interactions. Some examples are `EditText`, `Switch`, `CheckBox`. 
+
+ Some views are focusable by default and others are made focusable by having `focusable` set to `true`. 
+
+#### Accessibility Focus
+TalkBack is able to gain focus an all meaningful views of the screen which includes views that are non-focusable such as `ImageView` or `TextView`.
+
+To make a view focusable for accessible you can utilize : 
+* `android:importantForAccessibility` 
+*  In Android Pie and onwards,  `android:screenReaderFocusable`. 
+*  You can also make the view focusable by utilizing `setFocusable(boolean focusable)` in the `AccessibilityNodeInfo` of an `AccessibilityDelegate`. 
+
+#### Focus Guidelines
+* It is important that you manage how focus is being set. Ensure you don't confuse keyboard focus/input focus with accessibility focus, as you can end up in a situation where you set `focusable` to true for accessibility reasons and it impacts the user experience for users using the app without TalkBack. 
+* Do not attempt to force TalkBack to focus on a specific view, as it breaks predictable navigation and you might make the app more inaccessible. TalkBack also attaches itself late in the activity lifecyle so it might be difficult to get this to work properly without a hacky solution. 
+
+### <a name="horizontal-scrolling"></a>Horizontal Scrolling
+
+Horizontal scrolling views are common UI components in every developer's arsenal. In Android, we primarily utilize the `ViewPager` to create this experience. These views have to be optimized for TalkBack, since a visually impaired might find it difficult to know that it needs to be scrolled horizontally. To remedy this situation, visual indicators can be added so that TalkBack can scroll horizontally if necessary. This could be as simple as putting a Previous and Next button within the view or modifying the TalkBack announcement of the element, so that users know they can scroll for more content.
+
+
+### <a name="custom-views"></a>Custom Views & Behavior
+
+#### Custom Announcements
+If the Android framework is announcing something that needs more customization or a state change in the app needs to be made known to the user, the [`announceForAccessibility`](https://developer.android.com/reference/android/view/View.html#announceForAccessibility(java.lang.CharSequence)) method of the `view` can be utilized to specify this message. By default, the framework handles creating announcements for various events, and state changes well once the correct semantics are supplied. So this should only be utilized when necessary.
+
+#### Custom Events
+For full accessibility support within custom views, you have to hook the accessibility events within different the different actions and events that are occurring within that view. For full backward compatibility `ViewCompat.setAccessibilityDelegate()` can be utilized. 
```

Comment:
Just a small grammar nit here :) I am thinking the extra `different` added here was not intentional?

Reply:
It wasn't intentional. Thank you so much for catching this 😅